src/index.js

```diff
@@ -1 +1,50 @@
 import './style.css';
+
+const name = document.querySelector('.name');
+const score = document.querySelector('.score');
+const submitBtn = document.querySelector('.submitBtn');
+const refreshBtn = document.querySelector('.refreshBtn');
+const scoreList = document.querySelector('.scoreList');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/aJpln9nj3naXwqc1haiG/scores/';
+
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify({
+      user: name.value,
+      score: score.value,
+    }),
+
+  })
+    .then((response) => response.json())
+    .then((json) => json).catch((error) => error);
```

Comment:
- [ ] (Required) One of the project requirements is for you to use async and await to send and receive data. You have done well using async and await to receive data, but you need to use async and await to post data also. Kindly implement it 👍 . You can read this post from stack overflow to have an idea. https://stackoverflow.com/questions/50046841/proper-way-to-make-api-fetch-post-with-async-await

Reply:
@Gambit142 Thank you so much for the hint, I really appreciate it.